src/components/Loading.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from '../styles/loading'
+
+import React from 'react'
+
+const Loading = ({ message }) => {
+  return (
+    <div className={styles['fil-loading']}>
+      <p>{message}</p>
+    </div>
+  )
+}
+
+export default Loading
```

Comment:
I think maybe it's better to put the translated message as argument instead of include the translation library here in this little component.